pkg/policy/consumer.go

```diff
@@ -184,7 +184,11 @@ func (c *Consumable) addToMaps(id NumericIdentity) {
 }
 
 func (c *Consumable) wasLastRule(id NumericIdentity) bool {
-	return c.ReverseRules[id] == false && c.IngressIdentities[id] == false
+	// A rule is the 'last rule' for an identity if it does not exist as a key
```

Comment:
Make this comment as part of wasLastRule's documentation

Reply:
Fixed in latest upload.